.Documentation/articles/innoactive-creator/training-configuration.md

```diff
@@ -18,12 +18,12 @@ In the `Configuration` dropdown menu you can select the `Runtime Configuration`
 
 In the `Selected Training Course` dropdown menu you can select the training course that should be used in this scene.
 
-## Open Course in Workflow Editor
+## Open Course in Workflow window
 
-This button opens the `Workflow Editor` if it is not already open. It also displays the currently selected training course for editing.
+This button opens the `Workflow window` if it is not already open. It also displays the currently selected training course for editing.
 
-> Learn more about the [Workflow Editor](workflow-editor.md).
+> Learn more about the [Workflow window](workflow-editor.md).
```

Comment:
```suggestion
Learn more about the Workflow window [here](workflow-editor.md).
```

Reply:
I would not add the here, people understand that it is a link like this too